common/src/main/java/gov/cms/ab2d/common/health/DatabaseAvailable.java

```diff
@@ -0,0 +1,26 @@
+package gov.cms.ab2d.common.health;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.Statement;
+
+/**
+ * Does a health check on a database
+ */
+public class DatabaseAvailable {
+    /**
+     * Returns true if you can connect to the database and do a simple select
+     *
+     * @param datasource - the datasource to test
+     * @return true if you can access the db, false otherwise
+     */
+    public static boolean isDbAvailable(DataSource datasource) {
+        try (Connection conn = datasource.getConnection()) {
+            Statement stmt = conn.createStatement();
+            stmt.execute("SELECT 1");
+            return true;
+        } catch (Exception ex) {
+            return false;
+        }
+    }
```

Comment:
Need to close both the statement and connection in a `finally` clause to avoid pool leaks.

Reply:
Doesn't the try with resource close the connection?

Reply:
I think you need both the statement and connection in the try (...) section

Reply:
@lailasharshar My bad.

Yeah, just add the statement to the `try` as Andy is suggesting